### PR TITLE
guides: add walkthroughs

### PR DESCRIPTION
**Pull request by craig-osterhout**
### Proposed changes

- Add walkthroughs based on learning center walkthroughs to start aligning experience
- Add temporary overview page for walkthroughs
- Add temporary callout in the Get started guide for the walkthroughs

### Related issues (optional)

ENGDOCS-1760

https://deploy-preview-18609--docsdocker.netlify.app/get-started/
https://deploy-preview-18609--docsdocker.netlify.app/guides/walkthroughs/

### CODE REVIEW

File: content/guides/walkthroughs/run-hub-images.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+You can search for Docker Hub images on Docker Desktop. To search for the image used in this walkthrough, do the following:
+
+1. Open Docker Desktop and select the search.
+2. Specify `docker/welcome-to-docker` in the search.
+
+![Search Docker Desktop for the welcome-to-docker image](images/getting-started-search.png?w=400)
+
+## Step 2: Run the image
+
+To run the `docker/welcome-to-docker` image, do the following:
```
Comment (aevesdocker) [inline content/guides/walkthroughs/run-hub-images.md:27]: Nit: do we need 'do the following'? we could just dive straight in with 'To run....image:'?

(I see this is repeated throughout the walkthroughs)

File: content/guides/walkthroughs/multi-container-apps.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+```console
+$ cd /path/to/multi-container-app/
+$ docker compose up -d
+```
+```console
+$ docker compose up -d
+```
+
+## Step 4: View the frontend and add todos
```
Comment (aevesdocker) [inline content/guides/walkthroughs/multi-container-apps.md:58]: could we add something about Compose Watch here?

**Review by aevesdocker — COMMENTED** [review]
Really great @craig-osterhout. Nice use of 'related information' sections as well. 

Just two small comments from me. 

Also, noticed all your images are pngs. Could these be changed to webp? I know it's a faff so can be done in a follow up. 

https://squoosh.app/ go-to tool for converting

File: content/guides/walkthroughs/multi-container-apps.md (hunk 2 above)
Comment (craig-osterhout) [reply content/guides/walkthroughs/multi-container-apps.md:58]: Yep, definitely! I was holding off until the sample app was updated, but it looks like they just merged it https://github.com/docker/multi-container-app/commit/7b11449d8a58a8d8dbb1a5954ed8a96b757a0b91

**Review by aevesdocker — APPROVED** [review]
:shipit:

**Pull request merged**